index.html

```diff
@@ -1821,15 +1829,29 @@
               </ol>
             </li>
           </ol>
+          <div class="note">
+            Multiple requests to run the algorithm to <a>monitor the list of
+            available presentation displays</a> may occur at once, e.g., due to
+            calls to <a for="PresentationRequest">getAvailability</a> and
+            <a for="PresentationRequest">start</a>. The <a>user agent</a> may
+            run such requests in parallel, serialize them, or interrupt the
+            algorithm if it is before or at step 5 and run it again to
+            aggregate requests. The <a>user agent</a> may also aggregate
+            monitoring activities across <a data-lt="browsing context">browsing
+            contexts</a>.
+          </div>
```

Comment:
It is clear and it reads better. I replaced the whole note with the suggested text.